app/src/main/java/com/example/criminalintentapp/presentation/fragments/crime_detail/CrimeFragment.kt

```diff
@@ -231,6 +245,23 @@ class CrimeFragment : Fragment(R.layout.fragment_crime), FragmentResultListener
         }
     }
 
+    private fun updateFirestore(){
```

Comment:
What happens to the image? If I remember correctly there is also an image that is saved locally. Does this also get saved on firebase?